index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal-Portfolio-Page</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <main>
+        <header class="header">
+            <nav class="nav row">
+                <ul class="nav-logo">
+                    <li class="logo-item"><img src="img/image_geometry_1.svg" alt="logo-image" class="logo-img"></li>
+                </ul>
```

Comment:
- [ ] Please kindly take note that semantically the logo supposed to be wrapped with a link `a` tag, doing this will make it a clickable link. Thanks. 🙏 
- [ ] Since you only have one element in the nav bar, do you think it is necessary to use the` ul and li` tags? I think you can remove it to make your work clean and readable. ℹ️ 
- [ ] A logo is not part of navigation links, kindly bring out the logo out of the nav tag. 👍 
```suggestion
                <a><img src="img/image_geometry_1.svg" alt="logo-image" class="logo-img"></a>
                            <nav class="nav row">
               
```